Bottom: e454e38dc780e6b0b1d7edbe5900ec6f695c2c83
Top:    1eee944048723bf3ba671f04d3b52fce37592559
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
